demos/src/accordion.mustache

```diff
@@ -0,0 +1,38 @@
+<div data-o-component="o-expander" class="o-expander items" data-o-expander-shrink-to="3" data-o-expander-count-selector="li">
+	<h2>Simple List</h2>
+	<ul class="o-expander__content">
+		<li>item</li>
+		<li>item</li>
+		<li>item</li>
+		<li>item</li>
+	</ul>
+	<a href='#' class="o-expander__toggle o--if-js"></a>
+</div>
+
+<div data-o-component="o-expander" class="o-expander items" data-o-expander-shrink-to="2" data-o-expander-count-selector="li" data-o-expander-accordion="true">
+	<h2>Accordion behaviour</h2>
+	<ul class="o-expander__content">
+		<a href='#' class="o-expander__toggle o--if-js" data-expand-id="list-1">List 1</a>
+		<span class="o-expander__content" data-id="list-1">
```

Comment:
They could, but I don't think `data-id` is overly useful when there's already `id`. That and the ability to link using the `href="#list-1"` on a link and a regular `id="list-1"` to me would make this a more useful approach.